src/views/ManageList.tsx

```diff
@@ -76,6 +85,44 @@ export function ManageList({ listPath }: Props) {
 		}
 	};
 
+	const handleInvite = async (
+		e: FormEvent<HTMLFormElement>,
+		listPath: string,
+	) => {
+		console.log("Button clicked! Inviting user!");
+		e.preventDefault();
+
+		const trimmedEmailName = validateTrimmedString(emailName);
+
+		if (!trimmedEmailName) {
+			toast.error(
+				"Oops! Email cannot be empty or just spaces. Please enter a valid email.",
+			);
+			return;
+		}
+
+		if (currentUser === null) {
+			toast.error(`You are not logged in! Cannot invite.`);
```

Comment:
I think it would be best to not render this component at all instead of notifying the user that they cannot do this operation after they have already attempted to do so

Reply:
Yes please! We were getting an odd message (that I can't remember right now), and that null-check solved the issue.
BUT, thinking about it now, we could just do a ternary operator like,

{currentUser && //code here}

right?

Reply:
Yup, that's correct! @bbland1 has been working to make this part of the codebase much more straight forward, so this shouldn't be an issue for much longer 😄 https://github.com/the-collab-lab/tcl-77-smart-shopping-list/pull/27/files

Reply:
![image](https://github.com/user-attachments/assets/dd008b54-d49a-40b4-b410-ec0b84d473c2)
Perfect! Just make it as clean as possible to save on extra refactoring and feelings later!